packages/react-components/react-avatar/src/components/AvatarGroupItem/useAvatarGroupItemStyles.ts

```diff
@@ -111,15 +70,8 @@ const useStackStyles = makeStyles({
     },
   },
   overflowButton: {
-    // border-color has to be set to transparent when there's focus due to the outline overlapping the focus ring.
-    '&:focus': {
-      '&::after': {
-        ...shorthands.borderColor('transparent'),
-      },
-    },
-    // hide inner border when using high contrast mode and use the outer (::after) to match Avatar's outline
-    '@media (forced-colors: active)': {
-      ...shorthands.borderColor('Canvas'),
```

Comment:
I removed it because it's not needed anymore thanks to the rework, here's how it looks:
<img width="210" alt="image" src="https://user-images.githubusercontent.com/5953191/182509537-95d9f4cf-e9b8-4b86-84e5-3021df07fd69.png">

<img width="246" alt="image" src="https://user-images.githubusercontent.com/5953191/182509618-d3abc044-fdcb-4dfa-bf08-e51a4646b93c.png">

<img width="213" alt="image" src="https://user-images.githubusercontent.com/5953191/182509678-a499a9be-3be4-41af-80b5-24f2999c3267.png">